test/new-e2e/tests/process/windows_test.go

```diff
@@ -147,5 +143,7 @@ func (s *windowsTestSuite) TestManualProcessCheckWithIO() {
 	check := s.Env().RemoteHost.
 		MustExecute("& \"C:\\Program Files\\Datadog\\Datadog Agent\\bin\\agent\\process-agent.exe\" check process --json")
 
-	assertManualProcessCheck(s.T(), check, true, "MsMpEng.exe")
+	// Check stats for Datadog agent process as it has IO stats more reliably populated than MsMpEng.exe
+	agentExe := "\"C:\\Program Files\\Datadog\\Datadog Agent\\bin\\agent.exe\""
+	assertManualProcessCheck(s.T(), check, true, agentExe)
```

Comment:
It's been a while so I could be misremembering, but I think the initial reasoning for not using the agent process in the test is we lose ability to capture whether the agent has the proper permissions to fetch these stats from other users than itself.

Reply:
Maybe @robertjli remembers.

Reply:
That should be covered by the fact that in the other tests we're checking non-agent processes, no?

Reply:
discussed in https://github.com/DataDog/datadog-agent/pull/22799#discussion_r1486858604